src/services/email.service.js

```diff
@@ -0,0 +1,65 @@
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: true,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+function send({ email, subject, html }) {
+  return transporter.sendMail({
+    from: 'Auth API',
+    to: email,
+    subject,
+    text: '',
+    html,
+  });
+}
+
+function sendActivationLink(email, token) {
+  const link = `${process.env.CLIENT_URL}/activate/${token}`;
```

Comment:
Ensure consistent use of `CLIENT_URL` and `CLIENT_HOST` environment variables. Consider standardizing on one variable to avoid confusion and potential configuration issues.